rubocop.yml.tt

```diff
@@ -23,6 +23,10 @@ Layout/HashAlignment:
     - table
     - key
 
+Layout/LineLength:
+  Exclude:
+    - "config/initializers/content_security_policy.rb"
```

Comment:
The vite installer adds a really long line to this file. Rather than try to correct the installer, just disable the line length check so that rubocop doesn't complain.